UI/LinkText.cpp

```diff
@@ -3,14 +3,21 @@
 #include "ClientUI.h"
 #include "CUIControls.h"
 #include "../Empire/Empire.h"
+#include "../Empire/Government.h"
+#include "../universe/BuildingType.h"
```

Comment:
probably don't need all these `#include`s and can rather do a bunch of forward declares like https://github.com/freeorion/freeorion/blob/master/util/VarText.cpp#L22-L37

Reply:
i do not quite get at first glance why Species is there both as an include as well as as forward declaration from only looking at those few highlighted lines and nothing else, but ok i'll try that

Reply:
seems like GetSpecies already comes from AppInterface.h - otherwise done.